kornia/nerf/data_utils.py

```diff
@@ -12,6 +13,14 @@
 RayGroup = Tuple[Tensor, Tensor, Optional[Tensor]]
 
 
+def _is_list_of_str(lst: Sequence[object]) -> TypeGuard[List[str]]:
```

Comment:
Maybe these functions should be in a generatl `utils`?

Reply:
Or as KORNIA_CHECK_